data/tests.yml

```diff
@@ -841,9 +841,10 @@
         direction: forward
         designated: motor_vehicle
 
-- description: bus:lanes:forward=designated|
+- description: "bus:lanes:forward=designated|"
   skip_python: true
   skip_kotlin: true
+  rust: false # see next test
```

Comment:
I hadn't really thought about using skipped tests enough. Good examples in this PR.